buildpack.md

```diff
@@ -808,17 +817,51 @@ The lifecycle MUST include all unmatched files in the app directory in any numbe
 ### Build Plan (TOML)
 
 ```toml
-[<dependency name>]
+[[provides]]
+name = "<dependency name>"
+
+[provides.metadata]
+# buildpack-specific data
+
+[[requires]]
+name = "<dependency name>"
+version = "<dependency version>"
+
+[requires.metadata]
```

Comment:
according to the [Contractual Build Plan RFC](https://github.com/buildpack/rfcs/blob/master/text/0005-contractual-build-plan.md), `requires` doesn't have a metadata section.

Reply:
Good catch (assuming you mean `provides`). Fixed :)